storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2178,6 +2261,16 @@ dberr_t dict_load_tables_from_space_id(space_id_t space_id, THD *thd,
     goto error;
   }
 
+  /* duplicate SDI can exists in version lower than MySQL 8.0.24 or if database
```

Comment:
duplicate SDI can exist in file_per_table IBD files lower MySQL version 8.0.24 if table is used in EXCHANGE PARTITION or IMPORT. Even after upgrade to latest version 8.0.25 (which fixed the duplicate SDI issue), such
tables continue to contain duplicate SDI. PXB will scan the DD table "mysql.tables" to determine correct SDI

Reply:
fixed